aspnetcore/blazor/javascript-interoperability/call-dotnet-from-javascript.md

```diff
@@ -404,9 +404,64 @@ Objects that contain circular references can't be serialized on the client for e
 * .NET method calls.
 * JavaScript method calls from C# when the return type has circular references.
 
+::: moniker range=">= aspnetcore-6.0"
+
+## Byte array support
+
+Blazor supports optimized byte array JS interop that avoids encoding/decoding byte arrays into Base64. The following example uses JS interop to pass a byte array to .NET.
+
+Inside the closing `</body>` tag of `wwwroot/index.html` (Blazor WebAssembly) or `Pages/_Host.cshtml` (Blazor Server), provide a `sendByteArray` JS function. The function is called by a button in the component and doesn't return a value:
+
+```html
+<script>
+  window.sendBytes = () => {
+    const data = new Uint8Array([0x45,0x76,0x65,0x72,0x79,0x74,0x68,0x69,
+      0x6e,0x67,0x27,0x73,0x20,0x73,0x68,0x69,0x6e,0x79,0x2c,
+      0x20,0x43,0x61,0x70,0x74,0x69,0x61,0x6e,0x2e,0x20,0x4e,
+      0x6f,0x74,0x20,0x74,0x6f,0x20,0x66,0x72,0x65,0x74,0x2e]);
+    DotNet.invokeMethodAsync('ByteArrayTesting', 'ReceiveByteArray', data)
```

Comment:
Perhaps

```suggestion
    DotNet.invokeMethodAsync('BlazorSample', 'ReceiveByteArray', data)
```

to keep consistent with other examples? Otherwise they'll have to create a new assembly this for this.